src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -61,14 +61,14 @@ void EnElfgrp_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnElfgrp* this = THIS;
     s32 sp24;
 
-    this->unk_147 = ENELFGRP_GET(&this->actor);
+    this->unk_147 = STRAY_FAIRY_TYPE(&this->actor);
```

Comment:
I think there's some confusion here; you only want to use `STRAY_FAIRY_TYPE` when you're dealing with an instance of Elforg. So for this, you want to use `ENELFGRP_GET`

```suggestion
    this->unk_147 = ENELFGRP_GET(&this->actor);
```